File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/ProcessingJobCreateAdjustProcessingJobRequestItemPackages.java
package io.github.thunkier.thunkmetrc.wrapper.models;
import com.fasterxml.jackson.annotation.JsonProperty;public class ProcessingJobCreateAdjustProcessingJobRequestItemPackages {
    @JsonProperty("Label")
    public String label;
    @JsonProperty("Quantity")
    public Integer quantity;
    @JsonProperty("UnitOfMeasure")
    public String unitOfMeasure;
}

File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/ProcessingJobCreateAdjustProcessingJobRequestItemPackages.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class ProcessingJobCreateAdjustProcessingJobRequestItemPackages(
    @JsonProperty("label")
    val label: String? = null,
    @JsonProperty("quantity")
    val quantity: Int? = null,
    @JsonProperty("unitOfMeasure")
    val unitOfMeasure: String? = null
)
